### ISSUE TITLE
Error in Layout/MultilineAssignmentLayout

### ISSUE THREAD

**Issue by maximeg** [rubocop-hq/rubocop#6088]
Just upgraded to 0.58.0 and I got an error with the Layout/MultilineAssignmentLayout cop.

## Expected behavior

No error.

## Actual behavior

Getting this error:

```
An error occurred while Layout/MultilineAssignmentLayout cop was inspecting [...]/app/models/concerns/app_scoped.rb:9:4.
undefined method `line' for nil:NilClass
/Users/maxime/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.0/lib/rubocop/cop/layout/multiline_assignment_layout.rb:59:in `check_new_line_offense'
/Users/maxime/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.0/lib/rubocop/cop/layout/multiline_assignment_layout.rb:52:in `check_assignment'
/Users/maxime/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.0/lib/rubocop/cop/mixin/check_assignment.rb:18:in `on_send'
/Users/maxime/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.0/lib/rubocop/cop/commissioner.rb:57:in `block (2 levels) in trigger_responding_cops'
```

## Steps to reproduce the problem

My file is:

```ruby
# frozen_string_literal: true

module AppScoped

  def self.included(base)
    base.belongs_to(:app, inverse_of: nil, required: true)

    # Creates a default_scope on the class
    base.send(     # this is line 9
      :default_scope,
      lambda do
        current_app = AppScope.current

        if current_app
          base.where(app_id: current_app.id)
        else
          raise(
            ::AppScope::ViolationError,
            "Trying to query an AppScoped model without a configured app"
          )
        end
      end
    )
  end

end
```

If I write `base.send(:default_scope,`, I still get the error.

Simplified:

```ruby
# error:
base.send(
  :default_scope, 
  -> {
    where(foo: "bar")
  }
)

# error:
base.send(:default_scope, -> {
  where(foo: "bar")
})

# no error:
base.send(:default_scope, -> { where(foo: "bar") })

## RuboCop version

```
$ rubocop -V
0.58.0 (using Parser 2.5.1.0, running on ruby 2.5.1 x86_64-darwin17)
```

**Comment by tisba** [comment]
This is another example where the cop currently errors:

```ruby
foo << items.map do |item|
  "#{item}!"
end
```

**Comment by koic** [comment]
Thanks for the feedback. I confirmed that and I opened a PR #6091.
